bin/python_scripts/solr_reindex_package_ids.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Set the FILENAME_BASE to the txt file containing the list of package_ids
+#
+# Execute script like this -
+# python solr_reindex_package_ids.py
+#
+
+import os
+import sys
+import subprocess
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+FILENAME_BASE = "reindex_datasets-package_ids_28-31_07"
+
+
+def setup_logging(log_to_file=False):
+    _format = logging.Formatter('%(asctime)s - %(levelname)s - %(message)s')
+    logger.setLevel(logging.INFO)
+    c_handler = logging.StreamHandler()
+    c_handler.setFormatter(_format)
+    logger.addHandler(c_handler)
+
+    if log_to_file:
+        f_handler = logging.FileHandler('%s.log' % FILENAME_BASE)
+        f_handler.setFormatter(_format)
+        logger.addHandler(f_handler)
+
+    logger.info('====================================================================')
+
+
+def reindex_solr():
+    with open("%s.txt" % FILENAME_BASE, "r+") as f:
+        lines = f.readlines()
+        for i, line in enumerate(lines):
+            paster_command = 'paster --plugin=ckan search-index rebuild {} --config /var/ckan/ckan.ini'.format(line.strip())
+
+            logger.info('CKAN reindex %d/%d - Running command: %s', i+1, len(lines), paster_command)
+
+            try:
+                subprocess.call(paster_command, shell=True)
```

Comment:
I think you'll want to use [`check_call`](https://docs.python.org/2/library/subprocess.html#subprocess.check_call) for it to raise an exception.

Reply:
ok, will change